scripts/tcl_commands/floorplan.tcl

```diff
@@ -157,7 +172,7 @@ proc place_io_ol {args} {
         --length $arg_values(-length)\
         {*}$flags_map(-unmatched_error)\
         {*}$arg_values(-extra_args)\
-        $arg_values(-def) |& tee [index_file $::env(floorplan_logs)/place_io_ol.log] $::env(TERMINAL_OUTPUT)
+        $arg_values(-def) |& tee  $::env(TERMINAL_OUTPUT)
```

Comment:
i think you missed `$log` here

Reply:
You're eagle-eyed, I did a scan for these and still ended up missing a couple. Thanks.